client/src/app/site/pages/meetings/services/media-manage.service.ts

```diff
@@ -48,16 +49,19 @@ export class MediaManageService {
 
     public constructor(
         private activeMeetingService: ActiveMeetingService,
-        private mediaAdapter: MeetingMediaAdapterService
+        private mediaAdapter: MeetingMediaAdapterService,
+        private mediaRepo: MediafileRepositoryService
     ) {
-        this.activeMeetingService.meetingObservable.subscribe(_ => {
-            for (const place of Object.keys(this.logoUrlSubjects)) {
-                (<any>this.logoUrlSubjects)[place].next(this.getLogoUrl(place as LogoPlace));
+        merge(this.activeMeetingService.meetingObservable, this.mediaRepo.getViewModelListUnsafeObservable()).subscribe(
```

Comment:
The `unsafe` prefix worries me, is it necessary here?

Reply:
Yes